README.md

```diff
@@ -25,14 +25,14 @@ If you need help using or installing Node.js, please use the
 * [Contributing to the project](./CONTRIBUTING.md)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
-### Community Resources
+### Unofficial Resources
 
 * IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
 see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
 
-*Please note that community provided resources are neither managed by -- nor
-necessarily endorsed by -- the Node.js TSC/CTC. Specifically, such resources are
-not currently covered by the [Node.js Moderation Policy][] and the selection and
+*Please note that unofficial resources are neither managed by -- nor necessarily
+endorsed by -- the Node.js TSC/CTC. Specifically, such resources are not
```

Comment:
Nit: Don't mention `TSC` or `CTC`. Just say `Node.js` or `the Node.js project` or `the Node.js maintainers` or `the Node.js Foundation` or even `Node.js governing bodies`.

`TSC` and `CTC` are jargon acronyms that someone doesn't have to bother with to get the message here.